src/functionalTest/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationTest.java

```diff
@@ -1686,4 +1691,63 @@ void testMiDataRequestSuccess() throws Exception {
 
         mockMvc.perform(request).andExpect(status().isOk());
     }
+
+    private boolean compareArtefacts(Artefact expectedArtefact, Artefact unexpectedArtefact) {
```

Comment:
You should be comparing expected artefact to the returned artefact. Can we rename the second parameter unexpectedToArtefact to returnedArtefact or something more meaningful please?

Nothing wrong with the logic here but the better way to compare objects will be adding the Lombok annotation @EqualsAndHashCode to the Artefact class. Then just compare using the standard "equals" for the 2 objects rather than member by member. Just an alternative solution but this is working as intended

Reply:
Updated the parameter name now :) 

I did consider the Equals annotation, however this method specific excludes some fields which we'd want to keep in non-test scenario's, therefore kept this seperate